### PR TITLE
fix(module:i18n): fix catalan translations

### PR DESCRIPTION
**Pull request by gpalomar**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
This pull request fixes the missing catalan translations.

Issue Number: [#2569](https://github.com/NG-ZORRO/ng-zorro-antd/issues/2569)

## Does this PR introduce a breaking change?
```
[ ] Yes
[X] No
```

## Other information

### CODE REVIEW

File: components/i18n/languages/pagination/ca_ES.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ export default {
   // Options.jsx
   items_per_page: '/ pàgina',
   jump_to: 'Anar a',
-  jump_to_confirm: 'Confirma',
+  jump_to_confirm: 'confirmar',
```
Comment (wendzhue) [inline components/i18n/languages/pagination/ca_ES.ts]: Does this need to be capitalized?

Comment (gpalomar) [reply components/i18n/languages/pagination/ca_ES.ts]: Yes

**Comment by gpalomar** [comment]
Why this pull request is not merged? I really need it.

**Comment by vthinkxie** [comment]
> Why this pull request is not merged? I really need it.

`confirmar` should be `Confirmar `

**Comment by wendzhue** [comment]
@gpalomar Got a minute to rebase and make the letter capitalized? ;)

**Comment by gpalomar** [comment]
I wrote it in lowercase because almost all languages has it in lowercase. I'm going to do an update replacing "confirma" with "Confirma" but be aware that only the following languages has that string capitalized: et_EE, fi_FI, it_IT.

**Pull request merged**